src/ds/d400/d400-private.h

```diff
@@ -161,7 +166,8 @@ namespace librealsense
             {RS405_PID, "5.12.11.8" },
             {RS455_PID, "5.13.0.50" },
             {RS457_PID, "5.13.1.1" },
-            {RS457_RECOVERY_PID, "5.13.1.1" }
+            {RS457_RECOVERY_PID, "5.13.1.1" },
+            {RS430_GMSL_PID, "5.16.1.111" },
```

Comment:
We will need the official FW version here?

Reply:
added TBD note to update once firmware is available.